layouts/auto_generated_product.tpl

```diff
@@ -119,7 +119,8 @@
 
     <script>
       if (template) {
-       template.handleProductPageContent();
+        template.handleProductPageContent();
+        template.handleProductImageClick("{{ product.id }}");
```

Comment:
Let's add this line only in edit mode. No need to present in live html body. Also no need to add quotes arround id (that is number).
```suggestion

        {%- if product and editmode %}
          template.handleProductImageClick({{ product.id }});
        {%- endif %}
```